pyartcd/pyartcd/pipelines/olm_bundle_konflux.py

```
@@ -99,30 +100,21 @@ async def olm_bundle_konflux(
-        if kubeconfig_env_var:
-            final_kubeconfig = os.environ.get(kubeconfig_env_var)
-
-        if not final_kubeconfig:
-            available_env_vars = list(GROUP_KUBECONFIG_MAP.values())
-            raise ValueError(
-                f"Kubeconfig required for Konflux builds. Provide --kubeconfig parameter or set one of: {', '.join(available_env_vars)}"
-            )
+        available_env_vars = list(PRODUCT_KUBECONFIG_MAP.values())
+        raise ValueError(
+            f"Kubeconfig required for Konflux builds. Provide --kubeconfig parameter or set one of: {', '.join(available_env_vars)}"
+        )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)